cmd/kubeadm/.import-restrictions

```diff
@@ -11,6 +11,7 @@ rules:
   - selectorRegexp: k8s[.]io/kubernetes
     allowedPrefixes:
       - k8s.io/kubernetes/cmd/kubeadm
+      - k8s.io/kubernetes/pkg/util/math
```

Comment:
/hold
kubeadm must not depend on kubernetes/pkg
you should remove the kubeadm changes from this PR.

leaving the other changes to component maintainers.
but once concern here is that this PR increases the binding between components and k/k/pkg.
instead these utilities should live in a separate repository.

Reply:
Hi neolit123, thanks for review!
You're right, I should move the `math` package to [kubernetes / utils](https://github.com/kubernetes/utils).
One thing to confirm is that I have to wait for the new package to be merged into kubernetes / utils, right?

Reply:
We also have https://github.com/kubernetes/component-helpers, which is an external repo (not `k/pkg`) where this may fit

> One thing to confirm is that I have to wait for the new package to be merged into kubernetes / utils, right?

You should be able to update `go.mod` to point to your branch to start work on the changes, if you wish, but if it doesn't get accepted that work could be moot

Reply:
Thank you very much anyway, I will try it the way you said.
Glad to learn new knowledge! ^-^